README.md

```diff
@@ -34,14 +37,37 @@ If you still want to use `cmd.exe` you will have to use a package manager like M
 Getting Started
 ---------------
 
-### Install
+### NPM install
+
+If you've got all the requirements above, you should be able to install the CLI directly from npm. If not,
+or if you have any trouble with this, please try the new standalone install steps just below.
 
 This might require elevated privileges in some environments.
 
 ```sh
 $ npm install --global --production resin-cli
 ```
 
+### Standalone install
+
+If you don't have node or a working pre-gyp environment, you can still install the CLI as a standalone
+binary. **This is in experimental and may not work perfectly yet in all environments**, but it seems to work
+well in initial cross-platform testing, so it may be useful, and we'd love your feedback if you hit any issues.
+
+To install the CLI as a standalone binary:
+
+* Download the latest zip for your OS from https://github.com/resin-io/resin-cli/releases.
+* Extract the contents, putting the `resin-cli` folder somewhere appropriate for your system (e.g. `C:/resin-cli`, `/usr/local/lib/resin-cli`, etc).
+* Add the `resin-cli` folder to your `PATH`. (
+[Windows instructions](https://www.computerhope.com/issues/ch000549.htm),
```

Comment:
From now and on, whenever I need this, I will visit the CLI docs instead of googling :)

Reply:
Just for the sake of comparison, here's the Etcher CLi instructions https://etcher.io/cli/
(although the Etcher CLI is officially still in beta, and I'm not sure how well tested it is)